Contribute.md

```diff
@@ -0,0 +1,32 @@
+# Contribute Guide
+
+this file contains about how to make a contribution
+
+## Whats Counts As A Good Logo
+
+    - The right shape
+    - The right business cues
+    - The right colors
+    - The right tone
+    - The right typography
+    - The right trends
+    - The right sizes
+
+    For More Details You Can See This Link [Here](https://99designs.com/blog/logo-branding/logo-characteristics/)
+
+## Instructions For Upload Your Logo
+
+We're always looking to add more logos to our collection. If you can design an original, beautiful logo, feel free to contribute.
+Take a look at https://www.librelogos.org/all/ to view examples of what we're looking for.
+
+- Fork and download the repo
+- Navigate to the folder where you've saved your code
+- Make a new branch
+- Upload your logo in this folder libre-loggos/src/images/logos/
+- Add Data About your logo in this file libre-loggos/src/data/logo.js For Example You Can Click This Link [Example](https://github.com/enjeck/libre-logos/blob/main/src/data/logos.js)
```

Comment:
libre-logos should have just one 'g'

Reply:
Don't use capital letters within sentences like this, except for proper nouns. E.g, instead of "You Can Click This Link", write "you can click this link"